packages/common/src/types/sqlRunner.ts

```diff
@@ -110,7 +110,7 @@ export type CreateSqlChart = {
     name: string;
     description: string | null;
     sql: string;
-    config: SqlRunnerChartConfig;
+    config: SqlRunnerChartConfig | {};
```

Comment:
When creating a SQL chart, do we want it to be undefined instead?
Because in TS `{}` represents any value that isn't `null` or `undefined`, so I think we can use something else, like a `null`?

Reply:
Good call. I wasn't quite sure the best way to go here. Now that I'm looking, it might actually be best to default it to the most minimum set of config

```
    metadata: {
        version: number;
    };
    type:  string;
```

Reply:
I will update

Reply:
oooh nice, unless you have to handle that in a lot of places, then an `undefined`/`null` is probably better - but I like the minimal set of config approach as well 👍

Reply:
I did this and it seems to work. I put it in the create call for now, but we might want to think about if the defaulting should happen somewhere else. Like, should the version number get generated in the BE? I think thats a separate change.